<img src="https://bioconductor.org/images/logo_bioconductor.gif" alt="BiocLogo">

# BioC Asia 2018

29-30 November, 2018  
University of Melbourne, VIC, Australia  

## General Information

Registration will be outside Room 253, Arts West, University of Melbourne

- Morning and Afternoon tea will be provided outside Room 253
- Lunch will not be provided and participants are required to make their own arrangements

Visitor Wi-Fi information:

- Username: `abacbs`
- Password: `N@a7co`

An [etherpad site](https://pad.carpentries.org/BiocAsia2018) is also available for any online discussion during the workshops

To initiate a virtual machine go to http://courses.bioconductor.org

## Workshop Day: 29 November, 2018

Rooms 261 & 263, Arts West, University of Melbourne

- All participants are expected to bring their own laptops
- WiFi access wll be through either eduroam or using the above login
- Access to power sockets will be available so please bring your power adapter
- Please ensure you are running a current browser (i.e. chrome, firefox etc)

### Provisional Timetable

| Time | Room 261 (Beginner) | Room 263 (Intermediate) |
|:--------------- |:----------------------------- |:----------------------------- |
| 9:00am - 10:30am | [*An Introduction To Linear Models*](https://monashbioinformaticsplatform.github.io/r-linear-abacbs2018/) <br> Paul Harrison | [*Building a Bioconductor Package*](https://github.com/Bioconductor/BiocIntro/blob/BiocAsia-2018/vignettes/A01-Package-Development.Rmd) <br> Martin Morgan |
| 10:30am - 11:00am | **Morning Tea** <br> (Provided) |
| 11:00am - 12:30pm | [*Genomic Ranges and plyranges*](https://sa-lee.github.io/plyranges/articles/common-tasks.html) <br> Stuart Lee |[*classifyR*](https://github.com/SydneyBioX/TrainClassifyR) <br> Dario Strbenac and Shila Ghazanfar |
| 12:30pm - 1:30pm | **Lunch**  | |
| 1:30pm - 3:00pm | [*Bioconductor Annotation Packages*](https://github.com/Bioconductor/BiocIntro/blob/BiocAsia-2018/vignettes/A02-Annotation.Rmd) <br> Martin Morgan | [*Single cell RNA-seq analysis*](http://bioinf.wehi.edu.au/edgeR/10X.pdf) <br> Yunshun Chen |
| 3:00pm - 3:40pm | [*Differential binding analysis for ChIP-seq data*](https://github.com/algarnham/ChIP-seq-workshop) <br> Alex Garnham | [*Differential analysis of HiC data with diffHic*](https://github.com/hcoughlan/diffHicTutorialBioCAsia2018) <br> Hannah Coughlan |

| 3:40pm - 4:10pm | **Afternoon Tea** <br> (Provided) | |
| 4:10pm - 5:00pm | *Differential binding analysis for ChIP-seq data* <br> (continued) | *Differential analysis of HiC data with diffHic* <br> (continued) |


## Conference Day: 30 November, 2018

Room 556, Arts West, University of Melbourne

### Provisional Timetable

| Time | Topic |
|:-------------- |:------------------------------------- |
| 9:00am - 9:50am | *Professor Martin Morgan* <br> [Project updates](https://docs.google.com/presentation/d/1X2vEGelcjJYvq046ltRhPiwZixs9OfdiTxQoI_iyg4E/edit?usp=sharing) & Getting to know you |
| 9:50am - 10:30am | **Lightning Talks** |
|              | *Mr Shian Su* (Walter & Eliza Hall Institute of Medical Research) <br> CellBench: software for comparing single cell analyses |
|              | *Dr Peter Hickey* (Walter & Eliza Hall Institute of Medical Research) <br> Using the Delayed Array framework for processing large datasets |
|              | *Mr Brian Skjerven* (Pawsey Super-Computing Centre) <br> The use of Docker containers to support R-based bioinformatics workflows on an HPC system |
|              | *Ms Anna Quaglieri* (Walter & Eliza Hall Institute of Medical Research) <br> R-Ladies: An update on the local and wider picture |
|              | *Dr Saskia Freytag* (Walter & Eliza Hall Institute of Medical Research) <br> dtangle: accurate and robust cell type deconvolution |
| 10:30am - 11:00am |  **Morning Tea** |
| 11:00am - 12:30pm | **Research talks / Software demos** (I) |
|            | *Dr Peter Hickey* (Walter & Eliza Hall Institute of Medical Research) <br> [Getting help and helping others (including future you)](abstracts/hickey.html) |
|            | *Mr Harry Fong* (CSL Ltd) <br> [abseqR: reporting and data analysis functionalities for Rep-Seq datasets of antibody libraries](abstracts/fong.html) |
|           | *Dr Tim Peters* (Garvan Institute of Medical Research) <br> [A general framework for evaluating cross-platform concordance in genomic studies](abstracts/peters.html) |
|           | *Prof. Gordon Smyth* (Walter & Eliza Hall Institute of Medical Research) | |
| 12:30pm - 1:30pm | **Lunch** |
| 1:30pm - 2:20pm  | **Keynote presentation** |
|                  | *Assistant Professor Stephanie Hicks* (Johns Hopkins University) <br> [Orchestrating Single-Cell RNA-Sequencing Analysis with Bioconductor](abstracts/hicks.html) | 
| 2:20pm - 3:40pm  | **Research talks / Software demos** (II) | 
|          | *Ms Sarah Williams* (Monash University) <br> [Celaref: Annotating single-cell RNAseq clusters by similarity to reference datasets](abstracts/williams.html) |
|          | *Dr Saskia Freytag* (Walter & Eliza Hall Institute of Medical Research) <br> [Efforts towards a definitive pipeline for the analysis of matched single cell nuclei RNA-seq.](abstracts/freytag.html) |
|          | *Ms Yingxin Lin* (University Of Sydney) <br> [scMerge: Integration of multiple single-cell transcriptomics datasets leveraging stable expression and pseudo-replication](abstracts/lin.html) |
|          | *Dr Davis McCarthy* (St Vincentâ€™s Institute of Medical Research) <br> [The ever-expanding and ever-improving Bioconductor ecosystem for single-cell data analysis](abstracts/mccarthy.html) |
| 3:40pm - 4:10pm | **Afternoon Tea** | 
| 4:10pm - 4:40pm | **Panel Discussion: Grand challenges in Single Cell Data Analysis** |
|           | *Moderator:* Associate Professor Alicia Oshlack |
|           | *Panelists:* <br> Assistant Professor Stephanie Hicks <br> Professor Martin Morgan |
| 4:40pm - 5:00pm | **Closing Remarks** |

