---
title: "Doing single-cell genetics with Bioconductor"
author:
- name: Davis McCarthy
  affiliation: St Vincent's Institute of Medical Research, Fitzroy, VIC, Australia
output:
  BiocStyle::html_document
---

# Abstract

Recent advances in single-cell genomics technologies have rightly spurred great excitement. Something of a gold rush has ensued for methods and papers exploring cell-to-cell variation in DNA sequence and gene expression. Due to the accessibility of the assays, as well as inherent interest, single-cell RNA-seq (scRNA-seq) has been by far the most widely exploited single-cell genomic technology and has proven a powerful way to gain new insights into heterogeneous cell populations. However, most single-cell studies to this point have ignored a very important axis of variation: genetics. In recent work, we have integrated genotypic and scRNA-seq data to explore the effects of common genetic variation on single-cell gene expression in development (single-cell QTLs) and to interrogate phenotypic differences between different clonal populations of cells tagged by somatic mutations (single-cell clonality). In this talk, I will discuss how these large-scale studies have been supported at every level by the Bioconductor ecosystem and the bevy of tools available for genetics and single-cell genomics. I will also introduce the cardelino package (github.com/PMBio/cardelino) for the mapping of single-cell transcriptomes to a clonal tree and soon-to-be-added to the Bioconductor toolbox for single-cell data analysis.